Zend/zend_ast.h

```diff
@@ -105,6 +106,7 @@ enum _zend_ast_kind {
 	ZEND_AST_GOTO,
 	ZEND_AST_BREAK,
 	ZEND_AST_CONTINUE,
+	ZEND_AST_PARTIAL_NAMED_ARG,
```

Comment:
Does this need a separate AST node? It would seem natural to me to represent this as NAMED_ARG containing a PARTIAL_ARG.